pkg/alertmanager/statefulset.go

```diff
@@ -624,6 +627,33 @@ func makeStatefulSetSpec(logger *slog.Logger, a *monitoringv1.Alertmanager, conf
 		configReloaderVolumeMounts = append(configReloaderVolumeMounts, configMount...)
 	}
 
+	if !version.LT(semver.MustParse("0.24.0")) {
```

Comment:
I think we have a GTE method
```suggestion
	if version.GTE(semver.MustParse("0.24.0")) {
```

Reply:
Ah! yes indeed, it's just that Simon once said he preferred this semantics so I have been sticking with it since.

Reply:
hmm I don't recall saying this but right now, I'd prefer @nicolastakashi suggestion. Sorry about the confusion!